lib/nimble_template/helpers/credo.ex

```diff
@@ -1,10 +1,75 @@
 defmodule NimbleTemplate.CredoHelper do
   alias NimbleTemplate.Generator
+  alias NimbleTemplate.Projects.Project
+
+  @do_single_expression_rule_name "CompassCredoPlugin.Check.DoSingleExpression"
+  @single_module_file_rule_name "CompassCredoPlugin.Check.SingleModuleFile"
 
   @spec disable_rule(String.t(), String.t()) :: :ok | {:error, :failed_to_read_file}
   def disable_rule(file_path, rule) do
-    Generator.prepend_content(file_path, """
-    # credo:disable-for-this-file #{rule}
-    """)
+    if File.exists?(file_path) do
+      Generator.prepend_content(file_path, """
+      # credo:disable-for-this-file #{rule}
+      """)
+    end
+  end
+
+  @spec disable_do_single_expression_rule(list()) :: :ok | {:error, :failed_to_read_file}
+  def disable_do_single_expression_rule(file_paths) do
+    Enum.each(file_paths, fn file_path ->
+      disable_rule(file_path, @do_single_expression_rule_name)
+    end)
+  end
```

Comment:
Instead of having a function for each rule that loops file_paths and calls `disable_rule` what about a more generic function `disable_rules` with an `s` that you pass the rule which calls `disable_rule` on each. Then we can just use:

`disable_rules(file_paths, @do_single_expression_rule_name)` 
and
`disable_rules(file_paths, @single_module_file_rule_name)` 

```elixir
project
    |> get_files_contain_multiple_modules()
    |> disable_single_module_file_rule()
```

vs

```elixir
project
    |> get_files_contain_multiple_modules()
    |> disable_rules(@do_single_expression_rule_name)
```
or something similiar

Reply:
Nice suggestion, refactored in https://github.com/nimblehq/elixir-templates/pull/308/commits/ce27c66a70f9c83889b6fe91e171cd5a03d244e4